pkg/cmd/attestation/api/client.go

```diff
@@ -130,6 +167,70 @@ func (c *LiveClient) getAttestations(url, name, digest string, limit int) ([]*At
 	return attestations, nil
 }
 
+func (c *LiveClient) fetchBundlesByURL(attestations []*Attestation) ([]*Attestation, error) {
+	fetched := make([]*Attestation, len(attestations))
+	g := errgroup.Group{}
+	for i, a := range attestations {
+		g.Go(func() error {
+			b, err := c.fetchBundleByURL(a)
```

Comment:
and then here, mimicing how the API/HTTPClients seem to named?, we could have a:

```suggestion
			b, err := c.GetBundle(url)
```

Reply:
(as a heuristic, I am inclined to see `func FooBarBas` paired with slightly differently named `func FooBarBa` as a source of confusion)